app/server/integration/meta-routes.test.ts

```diff
@@ -0,0 +1,34 @@
+import { getOrCreateServer } from '@local/core/server';
+import supertest from 'supertest';
```

Comment:
Yeah it's just because those are non grapql endpoints that supertest is needed. If you know of a built in library or better library that would do the same thing let me know.

Reply:
Yeah, I don't know of a better library or if there's a way to not need it.  I just think a comment would help explaining why because it seems confusing at first why it's required here, but not in the rest of the tests.